src/geometry/rtree.cpp

```diff
@@ -99,9 +99,12 @@ constexpr auto geometry_reporter = [](auto const& geom) -> std::size_t {
     const auto outer_size = geom.outer().get_allocator().report();
     // reduce allows for out-of-order execution of associative and commutative operation (std::plus)
     // transform to associative and commutative using a unary predicate (hole -> size_t)
-    return std::transform_reduce(std::execution::unseq, std::begin(inners), std::end(inners),
-                                 outer_size, std::plus{},
-                                 [](auto const& hole) { return hole.get_allocator().report(); });
+    return std::transform_reduce(
+#ifndef __APPLE__  // apple clang does not implement `std::execution` despite being a C++17 feature
```

Comment:
`__APPLE__` is only defined in clang? With gcc will use `std::execution` ?

Reply:
`__APPLE__` is defined in apple. do we use `gcc` to compile on apple?